### PR TITLE
Upgrade to bootstrap 3

### PR DESCRIPTION
**Pull request by snipe**
I went through and updated most of the interfaces to Bootstrap 3.1.1. 

The only other tweaks were:
- create a macro for the countries drop down menu and call that in the user form. 
- pulled the admin menu options into the primary nav and determine what's visible based on Sentry roles

(I had found the UI a little confusing going back and forth between admin and user views, and most modern UIs have them integrated.)

I still have a few forms left to polish (admin/users and admin/groups), but that shouldn't take long.

screen shot 2014-03-22 at 5 41 34 am
screen shot 2014-03-22 at 5 42 22 am

### CODE REVIEW

**Comment by snipe** [comment]
(I know it looks like a buttload of changes, but the vast majority are cosmetic.)

File: app/config/mail.php [hunk 1]
```diff
@@ -119,6 +119,6 @@
 	|
 	*/
 
-	'pretend' => false,
+	'pretend' => true,
```
Comment (snipe) [inline app/config/mail.php:122]: Sorry, this should probably not have been changed - slipped into my commits.

**Comment by brunogaspar** [comment]
Thanks for the effort, but as i mentioned here https://github.com/brunogaspar/laravel4-starter-kit/pull/49, i'm not accepting these kind of PR's as they have a lot of changes involved and would need time to review them and time is something i don't have atm.

Thanks for the effort anyway.

**Comment by snipe** [comment]
That's really unfortunate. Are you planning on upgrading this to bootstrap 3 any time soon? If not, I may just keep moving forward with my fork for those that need it. I saw #49, but hoped that since my changes were all cosmetic, it might not be as difficult to test and might make it through. Bootstrap 3 was released 7 months ago.

**Comment by brunogaspar** [comment]
Yes, it'll be updated, but only in may.

**Comment by peersbio** [comment]
Hi Bruno, 

Any plan to release the update version of laravel4-starter-kit?

Thanks

Andy

**Pull request closed without merging**